docs/development/developing-plugins.md

```diff
@@ -80,10 +80,10 @@ def fatman_runtime_env_vars(self) -> Optional[Dict[str, str]]:
 
 - `fatman_job_types` - Job types provided by this plugin
 ```python
-def fatman_job_types(self) -> Dict[str, Tuple[Path, Path]]:
+def fatman_job_types(self) -> dict[str, list[tuple[Path, Path]]]:
```

Comment:
Thinking out loud... this type is not very friendly, and it's used in many places. Yes, more complex are possible. Still it's a bit to read and analyze. What do you think about "typedefing" it (frequent C++ practice) and using through-out, ie. 
```
DictListImagesPath = dict[str, list[tuple[Path, Path]]]

def fatman_job_types(self) -> DictListImagesPath:
     ...
```
TBH, I'm not very proud of proposed name, maybe there's something more clever here.

Reply:
Yeah, I think I could use a typedef. However, this function is a public interface of the Plugin class, so I think it's good that you don't need to import any types when creating your plugin:
```python
from __future__ import annotations
from pathlib import Path

class Plugin:
    def fatman_job_types(self) -> dict[str, list[tuple[Path, Path]]]:
```
so that plugins remain independent of the Racetrack codebase.

Surely, I agree this isn't a friendly type, so I'll use a typedef at least in the internal module `job_type.py`. Is it a good trade-off?

Reply:
Good idea :)